admin/partials/nginx-helper-general-options.php

```diff
@@ -84,20 +114,20 @@
 				<table class="form-table">
 					<tr valign="top">
 						<td>
-							<input type="radio" value="enable_fastcgi" id="cache_method_fastcgi" name="cache_method" <?php checked( $nginx_helper_settings['cache_method'], 'enable_fastcgi' ); ?> />
+							<input type="radio" value="enable_fastcgi" id="cache_method_fastcgi" name="cache_method" <?php echo checked( $nginx_helper_settings['cache_method'], 'enable_fastcgi' ); ?> />
 							<label for="cache_method_fastcgi">
 								<?php
-								printf(
-									__( 'nginx Fastcgi cache (<a target="_blank" href="%s" title="External settings for nginx">requires external settings for nginx</a>)', 'nginx-helper' ),
-									esc_url( $nginx_setting_link )
-								);
+									printf(
+										wp_kses( '%1$s (<a target="_blank" href="%2$s" title="%3$s">%4$s</a>)', array( 'strong' => array(), 'a' => array( 'href' => array(), 'title' => array() ) ) ),
```

Comment:
I think escaping o/p of html code is make more sense her, not it's template/skeleton. Use wp_kses on sprintf o/p .